.github/workflows/labeler.yml

```
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)